docs/sphinx/contributing/testing_hpx.rst

```diff
@@ -25,6 +25,27 @@ After the tests have been built, you can invoke separate tests with the help of
 the ``ctest`` command. You can list all available test targets using ``make help
 | grep tests``. Please see the |ctest_doc|_ for further details.
 
+Running performance tests
+=========================
+We run performance tests on Piz Daint for each pull request using Jenkins. To
+run those performance tests locally or on daint, a script is provided under
+``tools/perftests_ci/local_run.sh`` (to be run in the build directory specifying
+the hpx source directory as the argument to the script, default is
+``$HOME/projects/hpx_perftests_ci``.
+
+Add new performance tests
+=========================
+To add a new performance test, you need to wrap the portion of code to benchmark
+with ``hpx::util::perftests_report``, passing the test name, the executor name
```

Comment:
```suggestion
with ``hpx::util::perftests_report``, passing the test name, the test name
```
?

Reply:
The table prints the test name and the executor name (see report above) for the stream benchmark in case we wanted to make the executor vary which was a part of the original plan I believe, do you want me to remove the second string parameter or change it to display something else?

Reply:
We discussed this offline. Since all the performance tests we have so far are parameterized on an executor we'll keep giving the executor name special status in the performance tests.